projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -257,6 +258,17 @@ export abstract class AbstractTaskPanelComponent extends AbstractPanelWithImmedi
     @Input()
     public set taskPanelData(data: TaskPanelData) {
         this._taskPanelData = data;
+        if (this._taskContentService instanceof UnlimitedTaskContentService && this.panelRef) {
+            this.collapse();
+            this._taskContentService.task = this._taskPanelData.task;
+            if (this._sub) {
```

Comment:
I know it is not the code of the author, and this is only for future reference: attribute name `_sub` does not express what is the purpose of this attribute. I searched up, that this attribute is a subscription of `changedFields`, so something like `_changedFieldSub` would me better.